### DISCUSSION TITLE
Feature: Support PostGIS data types

### DISCUSSION THREAD

**Discussion by eTopocartGitHub** [electric-sql/electric#557] [Feature requests]
Hello. Is there a plan or it is possible to support PostGIS data types, like `GEOMETRY`, etc?

**Comment by balegas** [comment]
This is part of our roadmap, but coming in a bit later.

**Comment by balegas** [comment]
Turning this into a feature request discussion

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
